x-pack/plugins/security_solution/public/management/cypress/e2e/rbac/endpoint_role_rbac_with_space_awareness.cy.ts

```diff
@@ -0,0 +1,113 @@
+  setKibanaPrivilegeSpace,
+  setRoleName,
+  setSecuritySolutionEndpointGroupPrivilege,
+} from '../../screens/stack_management/role_page';
+
+describe(
+  'When defining a kibana role for Endpoint security access with space awareness enabled',
+  {
+    tags: ['@ess', '@serverless', '@serverlessMKI'],
+    env: {
+      ftrConfig: {
+        productTypes: [{ product_line: 'security', product_tier: 'essentials' }],
+        kbnServerArgs: [
+          `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+            'endpointManagementSpaceAwarenessEnabled',
+          ])}`,
+          // Enable spaces for serverless
+          `--xpack.spaces.maxSpaces=10`,
+        ],
+      },
+    },
+  },
+  () => {
+    let spaceId: string = '';
+    const roleName = `test_${Math.random().toString().substring(2, 6)}`;
+
+    before(() => {
+      login(ROLE.system_indices_superuser);
+      createSpace(`foo_${Math.random().toString().substring(2, 6)}`).then((response) => {
+        spaceId = response.body.id;
+      });
+    });
+
+    after(() => {
+      if (spaceId) {
+        deleteSpace(spaceId);
+        spaceId = '';
+      }
+    });
+
+    beforeEach(() => {
+      login(ROLE.system_indices_superuser);
+      navigateToRolePage();
+      setRoleName(roleName);
+      openKibanaFeaturePrivilegesFlyout();
+      expandSecuritySolutionCategoryKibanaPrivileges();
+      expandEndpointSecurityFeaturePrivileges();
+    });
+
+    it('should allow configuration per-space', () => {
+      setKibanaPrivilegeSpace(spaceId);
+      setSecuritySolutionEndpointGroupPrivilege('all');
+      clickEndpointSubFeaturePrivilegesCustomization();
+      setEndpointSubFeaturePrivilege('endpoint_list', 'all');
+      setEndpointSubFeaturePrivilege('host_isolation', 'all');
+      clickFlyoutAddKibanaPrivilegeButton();
+      clickRoleSaveButton();
+
+      navigateToRolePage(roleName);
+      clickViewPrivilegeSummaryButton();
+      cy.getByTestSubj('expandPrivilegeSummaryRow').click({ multiple: true });
+
+      cy.getByTestSubj('privilegeSummaryFeatureCategory_securitySolution')
+        .findByTestSubj(`space-avatar-${spaceId}`)
+        .should('exist');
+
+      cy.get('#row_siem_expansion')
+        .findByTestSubj('subFeatureEntry')
+        .then(($element) => {
+          const features: string[] = [];
+
+          $element.each((_, $subFeature) => {
+            features.push($subFeature.textContent ?? '');
+          });
+
+          return features;
+        })
+        .should('deep.equal', [
+          'Endpoint ListAll',
+          'Trusted ApplicationsNone',
+          'Host Isolation ExceptionsNone',
+          'BlocklistNone',
+          'Event FiltersNone',
+          'Elastic Defend Policy ManagementNone',
+          'Response Actions HistoryNone',
+          'Host IsolationAll',
+          'Process OperationsNone',
+          'File OperationsNone',
+          'Execute OperationsNone',
+          'Scan OperationsNone',
```

Comment:
does this test that there is no `privilegesTooltip` displayed? why is the `description` which is present in the other rbac cy test missing?

(wanted to do a quick check locally in my browser on my branch, but interestingly i don't have any elements with `subFeatureEntry` test subject on a 2 days old kibana)

Reply:
Hi @gergoabraham  - thanks for the reivew

re: 
> why is the description which is present in the other rbac cy test missing?

This is the data retrieved from the Privilege summary flyout - not the Edit form. In that view, it displays the effective privileges for the role. To see this panel, click the "View privilege summary" link at the bottom of the role page:

<details><summary>Screen capture</summary>

<img width="1923" alt="image" src="https://github.com/user-attachments/assets/c6a54635-622f-4f63-8681-9666d20b3a3e">

</details>


Re:
> does this test that there is no privilegesTooltip displayed?

No, it does not. I had planned a test for it, but removed it because creating tests for this is kind of hard (lack of `data-test-subj`'s). I'll look at it again and see if I can put in another test that ensures the tooltips are not displayed

Reply:
Ok. I added tests to validate that the tooltip is not present. Thanks for the feedback